client/generator/src/main/java/io/openlineage/client/Generator.java

```diff
@@ -12,13 +14,13 @@
 public class Generator {
 
   public static void main(String[] args) throws JsonParseException, JsonMappingException, IOException {
-    String baseURL = "https://raw.githubusercontent.com/OpenLineage/OpenLineage/jsonschema/spec/OpenLineage.json";
-    File input = new File("../spec/OpenLineage.json");
+    String baseURL = "https://raw.githubusercontent.com/OpenLineage/OpenLineage/main/spec/OpenLineage.json";
```

Comment:
_Minor_: Would we always want to generate the model off `main`? or based on a release tag (ex: `0.1.0`)?

Reply:
We would want to generate from a tag. I use main in the meantime until we tag our first version